sale_service_project/models/sale.py

```diff
@@ -158,5 +167,21 @@ class SaleOrderLineTaskMaterials(models.Model):
     product_id = fields.Many2one(
         comodel_name='product.product', string='Material')
     quantity = fields.Float(
-        string='Quantity',
-        digits_compute=dp.get_precision('Product Unit of Measure'))
+        string='Quantity', digits=dp.get_precision('Product UoS'))
+
+
+class SaleOrderReport(models.AbstractModel):
+    _name = 'report.sale.report_saleorder'
+
+    @api.multi
+    def render_html(self, data=None):
+        report_obj = self.env['report']
+        report = report_obj._get_report_from_name('sale.report_saleorder')
+        docargs = {
+            'doc_ids': self._ids,
+            'doc_model': report.model,
+            'docs': self,
+            'formatLang': rml_parse(
```

Comment:
I know, but  the price is calculated into a variable in Qweb, so I have to use ```t-esc``` and ```formatLang``` for formatting.

Another approach is have the price computed in the work an material models. Maybe that way is better.